practice/timezone/timezone/logic.py

```diff
@@ -0,0 +1,45 @@
+"""timezone Logic."""
+
+from tzwhere import tzwhere
+import arrow
+
+
+def get_time_at_tz(tz_string):
+    """Return the current time (as a string) at a given timezone."""
+    utc = arrow.utcnow()
+    time_string = str(utc.to(tz_string))
+    return time_string
+
+
+def get_tz(input_string):
+    r"""Return the time zone given lat, long.
+
+    >>> get_tz('45,45')
+    'Europe/Moscow'
+    """
+    lon_lat_list = input_string.split(',')
+    if len(lon_lat_list) != 2:
+        raise IOError('needs two comma separated values.')
+    tz = tzwhere.tzwhere()
+    tz_sring = tz.tzNameAt(float(lon_lat_list[0]), float(lon_lat_list[1]))
+    return tz_sring
```

Comment:
Any code related to serialization or deserialization should live in the views, then the logic should just work on the native types. The core logic of this program deals with numeric lat and lngs, it doesn't care about how you happen to represent them in the URL.